apps/etterlatte-behandling/src/main/kotlin/behandling/tilbakekreving/TilbakekrevingService.kt

```diff
@@ -2,105 +2,53 @@ package no.nav.etterlatte.behandling.tilbakekreving
-                    perioder =
-                        listOf(
-                            KravgrunnlagPeriode(
-                                periode =
-                                    Periode(
-                                        fraOgMed = YearMonth.of(2023, 1),
-                                        tilOgMed = YearMonth.of(2023, 2),
-                                    ),
-                                skatt = BigDecimal(200),
-                                grunnlagsbeloep =
-                                    listOf(
-                                        Grunnlagsbeloep(
-                                            kode = KlasseKode(""),
-                                            type = KlasseType.YTEL,
-                                            bruttoUtbetaling = BigDecimal(1000),
-                                            beregnetNyBrutto = BigDecimal(1200),
-                                            bruttoTilbakekreving = BigDecimal(200),
-                                            nettoTilbakekreving = BigDecimal(200),
-                                            beloepSkalIkkeTilbakekreves = BigDecimal(200),
-                                            skatteProsent = BigDecimal(20),
-                                            resultat = null,
-                                            skyld = null,
-                                            aarsak = null,
-                                        ),
-                                        Grunnlagsbeloep(
-                                            kode = KlasseKode(""),
-                                            type = KlasseType.FEIL,
-                                            bruttoUtbetaling = BigDecimal(0),
-                                            beregnetNyBrutto = BigDecimal(0),
-                                            bruttoTilbakekreving = BigDecimal(0),
-                                            nettoTilbakekreving = BigDecimal(0),
-                                            beloepSkalIkkeTilbakekreves = BigDecimal(0),
-                                            skatteProsent = BigDecimal(0),
-                                            resultat = null,
-                                            skyld = null,
-                                            aarsak = null,
-                                        ),
-                                    ),
-                            ),
-                        ),
-                ),
-        )
-    }
+    fun hentTilbakekreving(tilbakerevingId: UUID): Tilbakekreving? =
+        inTransaction {
+            tilbakekrevingDao.hentTilbakekreving(tilbakerevingId)
+        }
 
     fun opprettTilbakekreving(kravgrunnlag: Kravgrunnlag) =
         inTransaction {
+            logger.info("Oppretter tilbakekreving=${kravgrunnlag.kravgrunnlagId} på sak=${kravgrunnlag.sakId}")
+
+            val sak =
+                sakDao.hentSak(kravgrunnlag.sakId.value) ?: throw Exception(
```

Comment:
Ref. kommentar fra Ingleiv lenger opp. Bedre å bruke f.eks. `requireNotNull` med melding. Ikke kritisk, men gir litt bedre feilmeldinger.

Reply:
Enig. Gikk for en custom exception som returnerer feilmeldingen i http responsen.